lottie/src/main/java/com/airbnb/lottie/network/DefaultLottieNetworkFetcher.java

```diff
@@ -0,0 +1,19 @@
+package com.airbnb.lottie.network;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import androidx.annotation.NonNull;
+
+public class DefaultLottieNetworkFetcher implements LottieNetworkFetcher {
+
+  @Override
+  @NonNull
+  public LottieFetchResult fetchSync(@NonNull String url) throws IOException {
+    final HttpURLConnection connection = (HttpURLConnection) new URL(url).openConnection();
```

Comment:
Could this be HttpsUrlConnection?

Reply:
HttpUrlConnection was already in your library.  If we change it to HttpsUrlConnection, won't anything break?

Reply:
[HttpsUrlConnection](https://developer.android.com/reference/kotlin/javax/net/ssl/HttpsURLConnection) is also part of the Android SDK but it extends HttpUrlConnection so you're good here.